src/components/Main-Content.js

```diff
@@ -0,0 +1,131 @@
+import React, { Component } from "react";
+import "../stylesheets/main-content.css";
+
+class MainContent extends Component {
+  render() {
+    return (
+      <div id = "mainContent" className = "container-fluid main-content">
+      <div className = "row">
+        <div className = "sub-content col-lg- 12 col-md-12 col-sm-12">
+        <h4>Introductions</h4>
+          <p>
+          <b>Mathball</b>  is a JavaScript library for competitive programming, implementing optimized algorithms for faster execution.
+          </p>
+          <p>
+          JavaScript was never built while keeping competitive coding in mind; let us all agree on that. Back in 1995, Brendan Eich developed JavaScript only for adding interactivity to web pages like handling a mouse click. Today, we can build servers, games, mobile apps, IoT apps and even machine learning in the browser is possible with JavaScript.
+          </p>
+          <blockquote>
+          Any application that can be written in JavaScript, will eventually be written in JavaScript. -- Atwood &#39;s Law
+          </blockquote>
+          <p>Hence, I believe maybe its time for people who are well versed with this particular language to finally consider using it in the realm of competitive programming as well.</p>
+        </div>
+        </div>
+        <h4 style = {{textAlign: "center"}}>Features</h4>
+        <div className = "features row">
+
+          <div className = "sub-content  col-lg-4 col-md-4 col-sm-12">
+          <div class="card">
```

Comment:
I think creating a new `Card` component would be more readable.
Also, pass props or state to this `Card` component, which contains `Title of card` `Feature Array` `Links Array`
And, parse those arrays to form links, so it is more readable.

Also to add more links only state or props will need to be altered.